templates/internet-of-things/smart-city.html

```diff
@@ -14,108 +14,118 @@
   https://docs.google.com/document/d/1Zh-IkLDnaqmPr4TTeUZ3mtyY2_OGW4iElfntYWZyveY/edit#
 {% endblock meta_copydoc %}
 
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
 
-  <section class="p-strip--suru-topped is-bordered">
-    <div class="u-fixed-width">
-      <h1>Build the future with smart cities</h1>
-      <p class="p-heading--4">Enhance transport, utilities and infrastructure with IoT</p>
-      <p>
-        <a href="/internet-of-things/contact-us"
-           class="p-button--positive js-invoke-modal">Contact Canonical</a>
-      </p>
-      <div class="u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/c75046c4-Smart+City.svg",
+  {% call(slot) vf_hero(
+    title_text='Build the future with smart cities',
+    subtitle_text='Enhance transport, utilities<br /> and infrastructure with IoT',
+    layout='50/50-full-width-image'
+    ) -%}
+    {%- if slot == 'cta' -%}
+      <a href="#get-in-touch" class="p-button--positive js-invoke-modal">Contact Canonical</a>
```

Comment:
you should specify a path to a static pages for those that don't have JS enabled. But with JS enabled this should trigger opening a modal form.